charts/gardener/controlplane/charts/application/templates/apiservice-v1alpha1-security-gardener-cloud.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: apiregistration.k8s.io/v1
 kind: APIService
 metadata:
-  name: v1alpha1.authentication.gardener.cloud
+  name: v1alpha1.security.gardener.cloud
```

Comment:
Generally speaking yes, but it will be done later with different PR. At the current stage `CredentialsBinding` is unusable, so it makes no sense to grant access to any user of the Gardener explicitly.

Reply:
OK.